src/components/IncomingMessageList/ConversationPreviewModal.jsx

```diff
@@ -92,7 +101,7 @@ ConversationPreviewBody.propTypes = {
   conversation: PropTypes.object
 };
 
-class ConversationPreviewModal extends Component {
+export class _ConversationPreviewModal extends Component {
```

Comment:
why the `_` prefix?  This seems like something to avoid, especially on a variable we're exporting.  If this is the only exported, then we should `export default ConversationPreviewModal` at the bottom and then imports can go by the file name.